src/payment/strategies/adyenv2/adyenv2.mock.ts

```diff
@@ -0,0 +1,48 @@
+import { OrderRequestBody } from '../../../order';
+import { PaymentInitializeOptions } from '../../payment-request-options';
+
+import { AdyenClient, AdyenConfiguration } from './adyenv2';
+
+export function getAdyenClient(): AdyenClient {
+    return {
+        adyenCheckout: jest.fn(() => {
```

Comment:
I think we might omit the return here since arrow functions return the content by default and it is more developer-friendly to read.